samples/portable-sdks/python/fan-out-fan-in/README.md

```diff
@@ -0,0 +1,149 @@
+3. [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli) (if using a deployed Durable Task Scheduler)
+
+## Configuring Durable Task Scheduler
+
+There are two ways to run this sample:
+
+### Using the Emulator (Recommended)
+
+The emulator simulates a scheduler and taskhub in a Docker container, making it ideal for development and learning.
+
+1. Install Docker if it's not already installed.
+
+2. Pull the Docker Image for the Emulator:
+```bash
+docker pull mcr.microsoft.com/dts/dts-emulator:v0.0.6
+```
+
+3. Run the Emulator:
+```bash
+docker run --name dtsemulator -d -p 8080:8080 -p 8082:8082 mcr.microsoft.com/dts/dts-emulator:v0.0.6
+```
+Wait a few seconds for the container to be ready.
+
+Note: The example code automatically uses the default emulator settings (endpoint: http://localhost:8080, taskhub: default). You don't need to set any environment variables.
+
+### Using a Deployed Scheduler and Taskhub in Azure
+
+For production scenarios or when you're ready to deploy to Azure:
+
+1. Create a Scheduler using the Azure CLI:
+```bash
+az durabletask scheduler create --resource-group <testrg> --name <testscheduler> --location <eastus> --ip-allowlist "[0.0.0.0/0]" --sku-capacity 1 --sku-name "Dedicated" --tags "{'myattribute':'myvalue'}"
+```
+
+2. Create Your Taskhub:
+```bash
+az durabletask taskhub create --resource-group <testrg> --scheduler-name <testscheduler> --name <testtaskhub>
+```
+
+3. Retrieve the Endpoint for the Scheduler from the Azure portal.
+
+4. Set the Environment Variables:
+
+   Bash:
+   ```bash
+   export TASKHUB=<taskhubname>
+   export ENDPOINT=<taskhubEndpoint>
+   ```
+
+   PowerShell:
+   ```powershell
+   $env:TASKHUB = "<taskhubname>"
+   $env:ENDPOINT = "<taskhubEndpoint>"
+   ```
+
+## How to Run the Sample
+
+Once you have set up either the emulator or deployed scheduler, follow these steps to run the sample:
+
+1. First, activate your Python virtual environment (if you're using one):
+```bash
+python -m venv venv
+source venv/bin/activate  # On Windows, use: venv\Scripts\activate
+```
+
+2. Install the required packages:
+```bash
+pip install -r requirements.txt
+```
+
+3. Start the worker in a terminal:
+```bash
+python worker.py
+```
+You should see output indicating the worker has started and registered the orchestration and activities.
+
+4. In a new terminal (with the virtual environment activated if applicable), run the client:
+```bash
+python client.py [number_of_items]
+```
+You can optionally provide the number of work items as an argument. If not provided, 10 items will be used by default.
+
+## Understanding the Output
+
+When you run the sample, you'll see output from both the worker and client processes:
+
+### Worker Output
+The worker shows:
+- Registration of the orchestrator and activities
+- Status messages when processing each work item in parallel, showing that they're executing concurrently
```

Comment:
Not sure if this is relavent or an issue, but when running this multiple times, each time I've seen the output as follows:
INFO:__main__:Processing work item: 1
INFO:__main__:Processing work item: 2
INFO:__main__:Processing work item: 10
INFO:__main__:Processing work item: 9
INFO:__main__:Processing work item: 8
INFO:__main__:Processing work item: 7
INFO:__main__:Processing work item: 6
INFO:__main__:Processing work item: 5
INFO:__main__:Processing work item: 4
INFO:__main__:Processing work item: 3

The order is kinda weird, but again, I don't think it's a big issue or even an issue at all